perses/tests/test_hybrid_builder.py

```diff
@@ -99,6 +123,173 @@
 
 forcefield = app.ForceField('amber99sbildn.xml')
 
+def generate_solvated_hybrid_test_topology(mol_name="naphthalene", ref_mol_name="benzene"):
```

Comment:
Might still be good to add docstrings even for test methods, since others often need to understand what they do for review or reuse.